src/ansys/pyensight/core/utils/omniverse_dsg_server.py

```diff
@@ -510,16 +521,19 @@ def create_dsg_points(
         return part_stage_url
 
     def create_dsg_material(
-        self, stage, mesh, root_name, diffuse=[1.0, 1.0, 1.0, 1.0], variable=None
+        self, stage, mesh, root_name, diffuse=[1.0, 1.0, 1.0, 1.0], variable=None, mat_info={}

```

Comment:
If the JSON material representation is provided, use values in the resulting material dictionary to override the core protobuffer colors/etc.